api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl.java

```diff
@@ -121,6 +121,11 @@ public Observation toFhirResource(Obs observation) {
 		return obs;
 	}
 	
+	@Override
+	public Obs toOpenmrsType(Observation fhirObservation) {
+		return new Obs();
```

Comment:
Last thing. This should be `return toOpenmrsType(new Obs(), fhirObservation);` That way, it actually does something.

Reply:
Finished

Reply:
This time travis has taken long to cope up with building